client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -45,23 +44,28 @@ export default React.createClass( {
 			jetpack: false,
 			feature: false,
 			compact: false,
-			shouldDisplay: null
+			shouldDisplay: null,
+			site: null,
+			translate: identity,
 		};
 	},
 
 	handleClick() {
-		if ( this.props.event || this.props.feature ) {
-			analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', {
-				cta_name: this.props.event,
-				cta_feature: this.props.feature,
+		const { event, feature, onClick } = this.props;
+
+		if ( event || feature ) {
+			this.props.recordGoogleEvent( 'calypso_upgrade_nudge_cta_click', {
```

Comment:
Also note that we should use the tracks event here, since the original was a tracks call. (Automattic's analytics tool)
```javascript
import { recordTracksEvent } from 'state/analytics/actions';
//... + connect it below
const { recordTracksEvent: recordTracks } = this.props
```